### PR TITLE
Release 0.33.1

### PR DESCRIPTION
**Pull request by SeanTAllen**

### CODE REVIEW

File: content/blog/0.33.1-released.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+Before the next release, we will have a new tool [ponyup](https://github.com/ponylang/ponyup) for doing Pony installations on Linux. You can use ponyup now to install 0.33.1 as well as nightly Pony builds. However, please note that there is a breaking change coming with ponyup within the next couple of weeks. If you want to use ponyup to install 0.33.1 and report any issues might encounter using either [ponyup](https://github.com/ponylang/ponyup/issues) or [ponyc](https://github.com/ponylang/ponyc/issues), we'd welcome the feedback.
+
+We want to warn you, however, that in a couple of weeks, you will need to delete your existing ponyup installation and reinstall it. For this reason, we recommend Linux users hold off on installing Pony 0.33.1 unless they are willing to help beta-test ponyup or need the bug-fix detailed below.
+
+Windows and macOS users, nothing has changed about the installation process for you; however, heads up Windows users, within the next few months, changes are coming.
+
+## Fix nested tuple access regression introduced by #3304
```
Comment (EpicEric) [inline content/blog/0.33.1-released.md]:
```suggestion
## Fix nested tuple access regression introduced by [#3304](https://github.com/ponylang/ponyc/pull/3304)
```

File: content/blog/0.33.1-released.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+PR #3304 introduced a bug that caused segfaults with nested tuple access, such as:
```
Comment (EpicEric) [inline content/blog/0.33.1-released.md]:
```suggestion
[PR #3304](https://github.com/ponylang/ponyc/pull/3304) introduced a bug that caused segfaults with nested tuple access, such as:
```

File: content/blog/0.33.1-released.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
++++
+draft = false
+author = "seantallen"
+categories = [
+    "Release",
+]
+date = "2019-12-13T09:12:02-05:00"
+title = "0.33.1 released"
++++
+Pony version 0.33.1 is now available. The release features no breaking changes for user's Pony code. We recommend updating at your leisure.
```
Comment (EpicEric) [inline content/blog/0.33.1-released.md]:
```suggestion
Pony version 0.33.1 is now available. The release features no breaking changes for users' Pony code. We recommend updating at your leisure.
```

**Comment by EpicEric** [comment]
Adding DO NOT MERGE so that future me doesn't do it beforehand like the last time.

**Comment by SeanTAllen** [comment]
@EpicEric updated.

**Pull request merged**